src/token_enum.h

```diff
@@ -227,6 +227,7 @@ enum c_token_t
    CT_MACRO,            // a macro def
    CT_QUALIFIER,        // static, const, etc
    CT_EXTERN,           // extern
+   CT_DECLSPEC,         // __declspec
```

Comment:
It's way too specific to add as a token. If anything, it should be something about the storage class.